.github/workflows/cpp.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       uses: actions/checkout@v4
     - name: Check clang-format
       if: needs.check-changed-files.outputs.cpp == 'true'
-      run: if [[ $(find ./ \( -path ./icicle/build -prune -o -path ./**/target -prune -o -path ./examples -prune \) -iname *.h -or -iname *.cuh -or -iname *.cu -or -iname *.c -or -iname *.cpp | xargs clang-format --dry-run -ferror-limit=1 -style=file 2>&1) ]]; then echo "Please run clang-format"; exit 1; fi
+      run: ./scripts/format_all.sh . --check
```

Comment:
Can you also update the [hooks script](https://github.com/ingonyama-zk/icicle/blob/main/scripts/hooks/pre-push#L13-L28) to use this newer script

Reply:
done